.github/workflows/release.yml

```diff
@@ -1,5 +1,8 @@
 name: Create release
 on:
+  schedule:
```

Comment:
Potential issues:
* The `release` environment requires approval, so this might essentially do nothing.
* We don't run tests before releasing, so this could deploy something broken. I wrote the release workflow assuming that we want to release "no matter what" (e.g., some tests failed, and we acknowledge they're just flaky or some API is down).

Reply:
As you say, it'll 'block' at the env approval stage, which is the goal: It'll do an auto message in the channel. Maintenance guy can then do a quick check: is it green, nothing blocking, ok, just press the :shipit: button.